examples/README.md

```diff
@@ -95,19 +94,25 @@ Loads data directly from an URL.
 
 ## [From JSON spec](https://github.com/procyon-rs/vega_lite_3.rs/blob/master/examples/from_json_spec.rs)
 
-```
+```rust
 cargo run --example from_json_spec
 ```
 
 Load the chart directly from the JSON spec.
 
 <img src="https://raw.githubusercontent.com/procyon-rs/vega_lite_3.rs/master/examples/res/screens/diverging_stacked_bar_chart.png">
 
-# Miscellaneous
+```rust
+cargo run --example from_mixed_json_rust
+```
+
+Create a chart from existing json and add new data
+
+## Miscellaneous
```

Comment:
Miscellaneous title should stay at heading level 1 in markdown

Reply:
Do not agree, markdown linter check a rule that said that in a markdown file, you could have only one title of level 1, and it should be the first line.

Reply:
I changed it due to the markdown linter